src/app/app-modules/set-password/set-password.component.html

```
@@ -1,100 +1,103 @@
-<div class="support-wrapper">
-  <form #passwordFields="ngForm" class="support-card" autocomplete="off">
-    <header class="support-header">
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)